random/random.go

```diff
@@ -0,0 +1,32 @@
+// Package random provides utilities and functions for generating random data.
+package random
+
+import (
+	"bytes"
+	"crypto/rand"
+	"math/big"
+)
+
+// Character sets that you can use when passing into RandomString
+const Digits = "0123456789"
+const UpperLetters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+const LowerLetters = "abcdefghijklmnopqrstuvwxyz"
+const SpecialChars = "<>[]{}()-_*%&/?\"'\\"
+
+var Base62Chars = Digits + UpperLetters + LowerLetters
```

Comment:
Neat 👍

Reply:
Oh wait, I think with all the special characters, this is not base 62. At the least, the variable should be renamed. Note also that the special characters may limit reusability of this unique ID, as many identifiers and even passwords don't allow those characters, whereas almost everything supports alphanumeric.

Reply:
This doesn't have the special characters :)

Reply:
Oh... Right. So is that variable unused?

Reply:
This is a convenience variable. The usage pattern is that you can control what characters to use to generate the string. Examples:

```
// Only lower case chars + digits
random.RandomString(6, random.Digits + random.LowerLetters)

// alphanumerics + special chars
random.RandomString(6, random.Digits + random.LowerLetters + random.UpperLetters + random.SpecialChars)

// Only alphanumerics (base62)
random.RandomString(6, random.Base62Chars)
```

Reply:
Ohhh, I gotcha now. Perhaps add a test case using some of the other character sets to make that a bit clearer?

Reply:
Addressed in d9def43